tests/common/snappi_tests/snappi_fixtures.py

```diff
@@ -681,7 +681,6 @@ def _get_multidut_snappi_ports(line_card_choice, line_card_info):
                         if port["peer_port"] in asic_port_map[asic] and hostname in port['peer_device']:
                             port['asic_value'] = asic
                             port['asic_type'] = host.facts["asic_type"]
-                            port['duthost'] = host
```

Comment:
is this change needed?

Reply:
@sdszhang  - I thought this change came from @selldinesh  and is not my change. We can discuss this and check if this is required or not.

Reply:
I believe the current code is fine as it is. Let's keep it unchanged.

Reply:
@amitpawar12 can you remove this change from this PR?

Reply:
Hi @sdszhang, This code is still part of the code and I am not sure why it was added in first place. I would recommend that we raise a different PR to remove this line of the code. I am not sure why it was added, and impact in case we removed it.

Thanks,